[2021-12-21 12:50:02,773] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: rank_update.pull_from_git 2021-12-21T12:40:00+00:00 [queued]>
[2021-12-21 12:50:02,780] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: rank_update.pull_from_git 2021-12-21T12:40:00+00:00 [queued]>
[2021-12-21 12:50:02,780] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-12-21 12:50:02,780] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-12-21 12:50:02,780] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-12-21 12:50:02,787] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): pull_from_git> on 2021-12-21T12:40:00+00:00
[2021-12-21 12:50:02,788] {standard_task_runner.py:52} INFO - Started process 128825 to run task
[2021-12-21 12:50:02,792] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'rank_update', 'pull_from_git', '2021-12-21T12:40:00+00:00', '--job-id', '1213', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/rank_update.py', '--cfg-path', '/tmp/tmp9cwa5jou', '--error-file', '/tmp/tmpqkdxmddz']
[2021-12-21 12:50:02,792] {standard_task_runner.py:77} INFO - Job 1213: Subtask pull_from_git
[2021-12-21 12:50:02,829] {logging_mixin.py:104} INFO - Running <TaskInstance: rank_update.pull_from_git 2021-12-21T12:40:00+00:00 [running]> on host airflowvm
[2021-12-21 12:50:02,912] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=rank_update
AIRFLOW_CTX_TASK_ID=pull_from_git
AIRFLOW_CTX_EXECUTION_DATE=2021-12-21T12:40:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-21T12:40:00+00:00
[2021-12-21 12:50:02,912] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-12-21 12:50:02,912] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'git -C /home/airflow/AlgorithmStudy_211124 pull']
[2021-12-21 12:50:02,915] {subprocess.py:75} INFO - Output:
[2021-12-21 12:50:03,439] {subprocess.py:79} INFO - Already up to date.
[2021-12-21 12:50:03,440] {subprocess.py:83} INFO - Command exited with return code 0
[2021-12-21 12:50:03,470] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=rank_update, task_id=pull_from_git, execution_date=20211221T124000, start_date=20211221T125002, end_date=20211221T125003
[2021-12-21 12:50:03,488] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-12-21 12:50:03,530] {local_task_job.py:151} INFO - Task exited with return code 0
